client/src/components/login/LoginForm.js

```diff
@@ -32,7 +33,7 @@ export default function LoginForm({history}) {
         ref={register({
           required: {
             value: true,
-            message: 'Username is required to log in.',
```

Comment:
We do a lot more work up-front so that our lives are much easier later on when we're iterating on the application.

Here, we're moving these messages to a centralized file so that everything is in one place. Also open opportunity for language packs if we decide to support like Swedish , we can just swap our the entire message file.